src/pranitaPJun20/ValidPhoneNumber.java

```diff
@@ -0,0 +1,83 @@
+package pranitaPJun20;
+
+public class ValidPhoneNumber {
+	void checkValidPhoneNumber(String phoneNumber) {
+		int length = phoneNumber.length();
+		if (length < 10 || length > 14) {
+			System.out.println("\nPhone number " + phoneNumber + " having length "+length +"---> Invalid length number entered");
+		} 
+		else {
+			switch (length) {
+			case 10:
+				if (checkIsDigit(phoneNumber) == true && (phoneNumber.startsWith("7") || phoneNumber.startsWith("8")
+						|| phoneNumber.startsWith("9"))) {
+					System.out.println("\nPhone number " + phoneNumber + " having length "+length +" is a valid Phone Number");
+				} else {
+					System.out.println("\n" + phoneNumber + " is not a valid Phone Number.");
+				}
+				break;
+
+			case 11:
+					if (phoneNumber.startsWith("0") && checkIsDigit(phoneNumber) == true && (phoneNumber.charAt(1) =='7' || phoneNumber.charAt(1) =='8'
+							|| phoneNumber.charAt(1) =='9')) {
```

Comment:
Here you should use startsWith instaed of chatAt bcoz charAt() index might be change

Reply:
Changes done as suggested.